### PR TITLE
Add optional volumeMount and volume to deployment

### PR DESCRIPTION
**Pull request by khartahk**
- Allow specifying volumes and volumeMounts
- Update chart values.yaml example for: volumeMount and volume

### CODE REVIEW

**Comment by mtougeron** [comment]
Thanks for the contribution! Can you also bump up the version for the chart?

**Comment by khartahk** [comment]
Sure. Should this be a patch or a minor version?

**Comment by mtougeron** [comment]
> Sure. Should this be a patch or a minor version?

What you did is perfect.

File: charts/k8s-aws-ebs-tagger/templates/deployment.yaml [hunk 1]
```diff
@@ -83,6 +83,16 @@ spec:
               port: http
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+        {{- if .Values.volumeMounts }}
+        volumeMounts:
```
Comment (mtougeron) [inline charts/k8s-aws-ebs-tagger/templates/deployment.yaml:87]: The `volumeMounts` should be part of the container not the outer spec (2 spaces more). It should be at the same indentation level as ports, resources, etc, right?

Comment (khartahk) [reply charts/k8s-aws-ebs-tagger/templates/deployment.yaml]: I've fixed the indentation and combined the commits into one.

**Comment by mtougeron** [comment]
Looks great. I'll cut a release shortly.

**Review by mtougeron — APPROVED** [review]

**Pull request merged**